Documentation/articles/getting_started/0_getting_started.md

```diff
@@ -1,8 +1,6 @@
 # Getting Started
 
-This section walks you through the basics of MonoGame and helps you create your first game.
```

Comment:
I hate loosing this line...  it is perfect to explain what this is for.

Reply:
Ok, will revert